backend/src/main/java/com/huellapositiva/application/exception/EmailNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.huellapositiva.application.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class EmailNotFoundException extends RuntimeException {
+    public EmailNotFoundException(String message) {
+        super(message);
+    }
+}
```

Comment:
We should not create a new Exception to be thrown only in tests

Reply:
It's done